src/lazy/expanded/e_expression.rs

```diff
@@ -127,6 +127,7 @@ impl<'top, D: Decoder> EExpression<'top, D> {
                 environment = self.new_evaluation_environment()?;
                 MacroExpansionKind::Template(TemplateExpansion::new(template_ref))
             }
+            MacroKind::ToDo => todo!("system macro {}", invoked_macro.name().unwrap()),
```

Comment:
🗺️ All of the not-yet-implemented system macros now have the `MacroKind` `ToDo` and will panic if used. This allowed me to insert placeholders for the remaining macro definitions, which aligned the existing macro definitions to the expected addresses.